frontend/src/components/Navbar.jsx

```diff
@@ -2,14 +2,25 @@ import React, { useContext } from "react";
 
 import Login from "./LoginForm";
 import userContext from "../contexts/context";
-import { divide } from "lodash";
 
 function Navbar() {
-  let user = useContext(userContext);
+  let { isAdmin, language, changeLanguage } = useContext(userContext);
 
   return (
     <div>
-      {user.admin ? (
+      {language === "hebrew" && (
```

Comment:
Instead of having these specific conditions, just have a dropdown which will be a language selector.
Create a dropdown input component (with `options`, `value`, `onChange` props etc), and make the `value` the current language, and give the possible languages as `options`. The possible language keys should come from the BE and be available in the global context.